java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -411,6 +413,470 @@ public boolean isClosed() {
         return closed;
     }
 
+    /* ======================================================================
+     * Buckets Access And Operations
+     * ====================================================================== */
+
+    /**
+     * Creates a bucket in Manta.
+     *
+     * @param path The fully qualified path of the Manta bucket.
+     * @return true when bucket was created
+     * @throws IOException                                     If an IO exception has occurred.
+     * @throws MantaClientHttpResponseException                If a http status code {@literal > 300} is returned.
+     */
+    public boolean createBucket(final String path) throws IOException {
+        return createBucket(path, null);
+    }
+
+    /**
+     * Creates a bucket in Manta.
+     *
+     * @param rawPath The fully qualified path of the Manta bucket.
+     * @param rawHeaders Optional {@link MantaHttpHeaders}. Consult the Manta api for more header information.
+     * @return true when bucket was created
+     * @throws IOException                                     If an IO exception has occurred.
```

Comment:
You don't need to format the javadoc in this way. Other portions of javadoc may have this formatting, but we should slowly change them back to a way with less leading spaces.

Reply:
Noted.